foundation-extensions/identity-for-edge-network/api-reference.md

```diff
@@ -339,10 +339,19 @@ AEPIdentityItem *item = [[AEPIdentityItem alloc] initWithId:@"user@example.com"
 
 ## resetIdentities
 
-Clears all identities stored in the Identity extension and generates a new Experience Cloud ID \(ECID\) . Using this API does not remove the identifiers from the server-side User Profile Graph or Identity Graph.
+Clears all identities stored in the Identity extension and generates a new Experience Cloud ID \(ECID\). Using this API does not remove the identifiers from the server-side User Profile Graph or Identity Graph.
+
+This is a destructive action. Once an ECID is removed it cannot be reused. The new ECID generated by this API can impact metrics like unique visitors and cause duplication of user profiles and device profiles. To remove existing custom identifiers, use [`removeIdentity`](#removeidentity).
+
+Some example use cases for this API are: 
+- During debugging, to see how new ECIDs (and other identifiers paired with it) behave with existing rules and metrics.
+- A last-resort reset for when an ECID should no longer be used.
+
+This API is not recommended for:
+- Standard app lifecycle events (bootup/foreground/background)
```

Comment:
What did you mean by this one?

Reply:
I was thinking of addressing common pitfalls in terms of using this API incorrectly; do you know of any common mistakes customers have made in the past with using this API? 

I thought if they incorporated this mistakenly into regular app lifecycle callbacks, it would generate a new user profile etc. each time a user closes -> opens an app. Hopefully I'm making sense!

Reply:
no, haven't really seen this one. I can think about couple more use-cases we can list here:
- Resetting user's consent and privacy settings; see [Privacy and GDPR](../../resources/privacy-and-gdpr.md).
- Removing previously synced advertising identifier after the advertising tracking settings were changed by the user; use setAdvertisingIdentifier(link) API instead.

Reply:
ok, in that case I'll remove the standard app lifecycle point and add the ones you listed. thank you!